terraform/nixpkgs-tarballs.tf

```diff
@@ -286,3 +286,73 @@ resource "fastly_tls_subscription" "nixpkgs-tarballs" {
 output "nixpkgs-tarballs-managed_dns_challenge" {
   value = fastly_tls_subscription.nixpkgs-tarballs.managed_dns_challenge
 }
+
+# Create an S3 bucket for CloudTrail logs
+resource "aws_s3_bucket" "nixpkgs-tarballs-cloudtrail-logs" {
+  bucket = "nixpkgs-tarballs-cloudtrail-logs"
+  # We can potentially make this public for transparency?
+  # But first I want to see what the logs look like.
+  acl = "private"
+
+  versioning {
+    enabled = true
+  }
+}
+
+# Create a CloudTrail
+resource "aws_cloudtrail" "nixpkgs-tarballs" {
```

Comment:
We're using `aws_s3_bucket_logging` on the other buckets.

Is that change on purpose?

Reply:
We want only write-events and access logs gives us both.